send/ADConnector.pm

```diff
@@ -575,8 +575,10 @@ sub add_members_to_entry($$$$) {
 		if ($response) {
 			unless ($response->is_error()) {
 				ldap_log($service_name, "Group members added: " . $ad_entry->dn() . " | \n" . join(",\n", @$_));
+				print STDERR "Group members added: " . $ad_entry->dn() . " | \n" . join(",\n", @$_);
```

Comment:
I think we should only log errors to STDERR.

Reply:
Yes, that would make more sense, thanks.